src/hotspot/cpu/x86/c2_stubGenerator_x86_64_string.cpp

```diff
@@ -0,0 +1,1837 @@
+/*
+ * Copyright (c) 2023, 2024 Intel Corporation. All rights reserved.
```

Comment:
Is the 2023 year intentional? I don't know your policy, so you can just ignore this ;)

Reply:
I started this in November :-)